lib/OpenQA/Git.pm

```diff
@@ -22,15 +23,24 @@ sub config ($self, $args = undef) {
 }
 
 sub _prepare_git_command ($self, $args = undef) {
-    my $dir = $args->{dir} // $self->dir;
+    my $dir;
```

Comment:
why is this change necessary?

Reply:
I've been running into `t/16-utils-runcmd.t .. 1/? Not a HASH reference at /home/rrichardson/Code/oqa/openQA/lib/
OpenQA/Git.pm line 26.`earlier, which is why i added the check. I think however the `args` argument may not be needed at all here.

Reply:
Just for information: there should not be an error message like this when `$args` is undef. Only if it is not undef and not a hash ref.
At that point you were probably accidentally passing some other argument.